source/utils/events.js

```diff
@@ -1,52 +1,65 @@
+// @flow
+// $FlowFixMe
```

Comment:
more context please!

Reply:
Same as above, the line below (line 3) is this import statement:
```
import type { SyntheticMouseEvent, SyntheticTouchEvent, SyntheticEvent, Element } from 'react';
```

These types are the recommended way to type DOM events with React and were released v16.3-ish, but Flow doesn't recognize them yet.